### PR TITLE
Add controllers config for karmada-controller-manager

### PR DESCRIPTION
**Pull request by yanggangtony**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
Re-check pr fix. 
And i found there is a duplicate define for the controllers.
[https://github.com/karmada-io/karmada/pull/3240](https://github.com/karmada-io/karmada/pull/3240#issuecomment-1461610806)

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by RainbowMango — COMMENTED** [review]
/assign @Poor12

**Comment by Poor12** [comment]
Would you like to check it in kube-controller-manager together? I'd like to delete the config level.

**Comment by yanggangtony** [comment]
@Poor12 
Thanks for review.
Keep the same with ```karmada-controller-manager.yaml```.
Delete the config level , and move it to the outer controller level.

File: charts/karmada/values.yaml [hunk 1]
```diff
@@ -500,7 +496,7 @@ kubeControllerManager:
     rollingUpdate:
       maxUnavailable: 0
       maxSurge: 50%
-
+  controllers: namespace,garbagecollector,serviceaccount-token,ttl-after-finished,bootstrapsigner,csrapproving,csrcleaner,csrsigning
```
Comment (Poor12) [inline charts/karmada/values.yaml:499]: Add a blank line here.

Comment (yanggangtony) [reply charts/karmada/values.yaml:500]: Got it.

Comment (Poor12) [reply charts/karmada/values.yaml:500]: Sorry, I mean add a blank between it and the etcd config.

Comment (yanggangtony) [reply charts/karmada/values.yaml:499]: :smile::smile:
Rebased it.

**Pull request merged**